src/test/java/teammates/test/cases/ui/browsertests/AdminHomePageUiTest.java

```diff
@@ -237,23 +237,20 @@ private void testCreateInstructorAction() throws Exception {
         
         ______TS("new instructor can send reminder of sample course");
         instructorHomePage.loadInstructorHomeTab();
-        feedbacksPage = instructorHomePage.clickFeedbackSessionRemindLink("AHPUiT.instr1.gma-demo", 
-                                                                          "Second team feedback session");
+        feedbacksPage = instructorHomePage.clickFeedbackSessionRemindLink("AHPUiT.instr1.gma-demo", "Second team feedback session");
```

Comment:
Is there a reason for removing the line break for the sake of shortening the lines?